### ISSUE TITLE
Crash when destroying output

### ISSUE THREAD

**Issue by danieldg** [swaywm/sway#5804]
- **Sway Version:** c523aa623bc5e5b5897c0f337058aa7308871fef (plus local patches)

- **Description:**
Crash triggered by waking monitor from dpms sleep (which normally triggers an unplug/plug event).

- **Stack Trace:**
```
Program terminated with signal SIGSEGV, Segmentation fault.
#0  0x000000000041f96b in popup_damage (layer_popup=0x2e11540, whole=true) at ../sway/desktop/layer_shell.c:411
411             struct sway_output *output = wlr_output->data;
(gdb) bt full
#0  0x000000000041f96b in popup_damage (layer_popup=0x2e11540, whole=true) at ../sway/desktop/layer_shell.c:411
        popup = 0x27d7f80
        surface = 0x2e14570
        popup_sx = 3137
        popup_sy = 52
        ox = 3137
        oy = 52
        layer = 0x2dd9020
        wlr_output = 0x0
        output = 0x23827a0
#1  0x000000000041fa46 in popup_handle_unmap (listener=0x2e11570, data=0x23fdca0) at ../sway/desktop/layer_shell.c:425
        popup = 0x2e11540
#2  0x00007f58a64dad15 in wlr_signal_emit_safe (signal=0x23fdda8, data=0x23fdca0) at ../subprojects/wlroots/util/signal.c:29
        pos = 0x2e11570
        l = 0x2e11570
        cursor = {link = {prev = 0x2e11570, next = 0x7fffe1733900}, notify = 0x7f58a64dac6e <handle_noop>}
        end = {link = {prev = 0x7fffe1733920, next = 0x23fdda8}, notify = 0x7f58a64dac6e <handle_noop>}
#3  0x00007f58a64aedae in unmap_xdg_surface (surface=0x23fdca0) at ../subprojects/wlroots/types/xdg_shell/wlr_xdg_surface.c:39
        __PRETTY_FUNCTION__ = "unmap_xdg_surface"
        popup = 0x23fdcd0
        popup_tmp = 0x23fdcd0
        configure = 0x2dd9020
        tmp = 0x28884d0
#4  0x00007f58a64afcc8 in reset_xdg_surface (xdg_surface=0x23fdca0) at ../subprojects/wlroots/types/xdg_shell/wlr_xdg_surface.c:474
#5  0x00007f58a64b00ea in wlr_xdg_popup_destroy (surface=0x23fdca0) at ../subprojects/wlroots/types/xdg_shell/wlr_xdg_surface.c:565
        __PRETTY_FUNCTION__ = "wlr_xdg_popup_destroy"
        popup = 0x23fdcd0
        popup_tmp = 0x23fdcd0
#6  0x00007f58a64bfc2c in layer_surface_unmap (surface=0x2ecd290) at ../subprojects/wlroots/types/wlr_layer_shell_v1.c:214
        popup = 0x27d7f80
        popup_tmp = 0x2ecd2a8
        configure = 0x428f4a <input_manager_get_seat+74>
        tmp = 0xffffffff
#7  0x00007f58a64bffa3 in wlr_layer_surface_v1_close (surface=0x2ecd290) at ../subprojects/wlroots/types/wlr_layer_shell_v1.c:298
#8  0x000000000041f363 in handle_output_destroy (listener=0x2dd9098, data=0x2cca430) at ../sway/desktop/layer_shell.c:280
        sway_layer = 0x2dd9020
        seat = 0x23827a0
        client = 0x2e8fde0
        set_focus = false
#9  0x0000000000469d30 in workspace_add_gaps (ws=0x2dd9020) at ../sway/tree/workspace.c:757
        total_gap = 0
        top_gap_frac = 6.9533304848963698e-310
#10 0x000000000046a952 in evacuate_sticky (old_ws=0x2cca430, new_output=0xb5aa3) at ../sway/tree/output.c:167
        sticky = 0x1c848d0
        new_ws = 0x2a406bc00
        __PRETTY_FUNCTION__ = "evacuate_sticky"
#11 0x0000000000422229 in damage_handle_destroy (listener=0x2cca598, data=0x2f97230) at ../sway/desktop/output.c:791
        output = 0x2cca430
#12 0x00007f58a64dad15 in wlr_signal_emit_safe (signal=0x2f972a0, data=0x2f97230) at ../subprojects/wlroots/util/signal.c:29
        pos = 0x2cca598
        l = 0x2cca598
        cursor = {link = {prev = 0x2cca598, next = 0x7fffe1733b60}, notify = 0x7f58a64dac6e <handle_noop>}
        end = {link = {prev = 0x7fffe1733b80, next = 0x2f972a0}, notify = 0x7f58a64dac6e <handle_noop>}
#13 0x00007f58a64c30ac in wlr_output_damage_destroy (output_damage=0x2f97230) at ../subprojects/wlroots/types/wlr_output_damage.c:130
#14 0x00007f58a64c2bd6 in output_handle_destroy (listener=0x2f972b0, data=0x285baa0) at ../subprojects/wlroots/types/wlr_output_damage.c:13
        output_damage = 0x2f97230
#15 0x00007f58a64dad15 in wlr_signal_emit_safe (signal=0x285bcd8, data=0x285baa0) at ../subprojects/wlroots/util/signal.c:29
        pos = 0x2f972b0
        l = 0x2f972b0
        cursor = {link = {prev = 0x2f972b0, next = 0x2cca538}, notify = 0x7f58a64dac6e <handle_noop>}
        end = {link = {prev = 0x306f6c8, next = 0x285bcd8}, notify = 0x7f58a64dac6e <handle_noop>}
#16 0x00007f58a648929e in drm_connector_cleanup (conn=0x285baa0) at ../subprojects/wlroots/backend/drm/drm.c:1623
        mode = 0x285bb40
        tmp = 0x285bb40
#17 0x00007f58a6488916 in scan_drm_connectors (drm=0x1c15200) at ../subprojects/wlroots/backend/drm/drm.c:1429
        drm_conn = 0x2444300
        c = 0x285baa0
        wlr_conn = 0x285baa0
        curr_enc = 0x2f27f00
        index = 0
        i = 0
        res = 0x2887f50
        seen_len = 4
        seen = {true, false, false, false, false}
        new_outputs_len = 0
        new_outputs = {0x7fffe1733cd0, 0x7fffe1734000, 0x7fffe1733ec0, 0x7fffe1733d00, 0xffffffff}
        conn = 0x2c6f834
        tmp_conn = 0x678984a
        index = 744066
#18 0x00007f58a6483bc3 in drm_invalidated (listener=0x1c152c0, data=0x1c0e970) at ../subprojects/wlroots/backend/drm/backend.c:115
        drm = 0x1c15200
        name = 0x2efdb50 "@x\346\002"
#19 0x00007f58a64dad15 in wlr_signal_emit_safe (signal=0x1c084d0, data=0x1c0e970) at ../subprojects/wlroots/util/signal.c:29
        pos = 0x1c152c0
        l = 0x1c152c0
        cursor = {link = {prev = 0x1c152c0, next = 0x7fffe1733f10}, notify = 0x7f58a64dac6e <handle_noop>}
        end = {link = {prev = 0x7fffe1733f30, next = 0x1c084d0}, notify = 0x7f58a64dac6e <handle_noop>}
#20 0x00007f58a649c131 in udev_event (fd=6, mask=1, data=0x1c0e970) at ../subprojects/wlroots/backend/session/session.c:57
        session = 0x1c0e970
        udev_dev = 0x2b32f90
        action = 0x7f58a5b7b590 "change"
        devnum = 57856
        dev = 0x1c084c0
#21 0x00007f58a5e2dac2 in wl_event_loop_dispatch () at /lib64/libwayland-server.so.0
#22 0x00007f58a5e2e1a5 in wl_display_run () at /lib64/libwayland-server.so.0
#23 0x000000000041d5ef in server_run (server=0x492360 <ipc_display_destroy>) at ../sway/server.c:247
#24 0x000000000041c509 in main (argc=2, argv=0x7fffe1734338) at ../sway/main.c:412
        verbose = 0
        debug = 0
        validate = 7
        allow_unsupported_gpu = 8
        long_options = {{name = 0x4744dd "", has_arg = 1, flag = 0x0, val = 99}, {name = 0x4744e4 "", has_arg = 0, flag = 0x0, val = 67}, {name = 0x4744ed "config", has_arg = 0, flag = 0x0, val = 100}, {name = 0x4744f3 "", has_arg = 0, flag = 0x0, val = 118}, {name = 0x4744fb "e", has_arg = 0, flag = 0x0, val = 86}, {name = 0x474503 "version", has_arg = 0, flag = 0x0, val = 112}, {name = 0x474512 "", has_arg = 0, flag = 0x0, val = 117}, {name = 0x474522 "unsupported-gpu", has_arg = 0, flag = 0x0, val = 117}, {name = 0x0, has_arg = 0, flag = 0x0, val = 0}, {name = 0x475b80 <buttons+32> "\024\001", has_arg = 1, flag = 0x475b86, val = 2}}
        config_path = 0x0
        usage = 0x473ea8 "debug flag: %s"
        c = -1
(gdb) p *layer
$2 = {layer_surface = 0x2ecd290, link = {prev = 0x2dd9028, next = 0x2dd9028}, destroy = {link = {prev = 0x2ecd398, next = 0x2ecd398}, notify = 0x41f661 <handle_destroy>}, map = {link = {prev = 0x2ecd3a8, next = 0x2ecd3a8},
    notify = 0x41f792 <handle_map>}, unmap = {link = {prev = 0x2ecd3b8, next = 0x2ecd3b8}, notify = 0x41f839 <handle_unmap>}, surface_commit = {link = {prev = 0x23fd3b0, next = 0x23fd3b0}, notify = 0x41f366 <handle_surface_commit>},
  output_destroy = {link = {prev = 0x0, next = 0x0}, notify = 0x41f286 <handle_output_destroy>}, new_popup = {link = {prev = 0x2ecd3c8, next = 0x2ecd3c8}, notify = 0x41fcf8 <handle_new_popup>}, geo = {x = 0, y = 0, width = 3840,
    height = 48}, layer = ZWLR_LAYER_SHELL_V1_LAYER_BOTTOM}
(gdb) p *layer->layer_surface
$3 = {surface = 0x23fd080, output = 0x0, resource = 0x24d9150, shell = 0x2330020, popups = {prev = 0x27d7f88, next = 0x27d7f88}, namespace = 0x2e7f9c0 "waybar", added = true, configured = true, mapped = true, closed = true,
  configure_serial = 429040, configure_next_serial = 429040, configure_list = {prev = 0x2ecd2d8, next = 0x2ecd2d8}, acked_configure = 0x0, client_pending = {anchor = 13, exclusive_zone = 48, margin = {top = 0, right = 0, bottom = 0,
      left = 0}, keyboard_interactive = false, desired_width = 0, desired_height = 48, actual_width = 0, actual_height = 0, layer = ZWLR_LAYER_SHELL_V1_LAYER_BOTTOM}, server_pending = {anchor = 0, exclusive_zone = 0, margin = {top = 0,
      right = 0, bottom = 0, left = 0}, keyboard_interactive = false, desired_width = 0, desired_height = 0, actual_width = 3840, actual_height = 48, layer = ZWLR_LAYER_SHELL_V1_LAYER_BACKGROUND}, current = {anchor = 13,
    exclusive_zone = 48, margin = {top = 0, right = 0, bottom = 0, left = 0}, keyboard_interactive = false, desired_width = 0, desired_height = 48, actual_width = 3840, actual_height = 48, layer = ZWLR_LAYER_SHELL_V1_LAYER_BOTTOM},
  surface_destroy = {link = {prev = 0x2ca40c8, next = 0x23fd3d0}, notify = 0x7f58a64c02b8 <handle_surface_destroyed>}, events = {destroy = {listener_list = {prev = 0x2dd9038, next = 0x2dd9038}}, map = {listener_list = {prev = 0x2dd9050,
        next = 0x2dd9050}}, unmap = {listener_list = {prev = 0x2dd9068, next = 0x2dd9068}}, new_popup = {listener_list = {prev = 0x2dd90b0, next = 0x2dd90b0}}}, data = 0x2dd9020}
```
- **Debug Log:**
Note: the entire log is 144MB.
```
206:41:04.829 [sway/commands.c:258] Handling command 'output * dpms on'
206:41:04.829 [sway/commands.c:426] Subcommand: dpms on
206:41:04.829 [sway/config/output.c:133] Merging output * config on HDMI-A-2
206:41:04.829 [sway/config/output.c:133] Merging output * config on HDMI-A-3
206:41:04.829 [sway/config/output.c:133] Merging output * config on DP-1
206:41:04.829 [sway/config/output.c:205] Merging on top of existing output config
206:41:04.829 [sway/config/output.c:228] Config stored for output * (enabled: -1) (-1x-1@-1.000000Hz position -1,-1 scale -1.000000 subpixel unknown transform -1) (bg (null) (null)) (dpms 1) (max render time: -1)
206:41:04.829 [sway/config/output.c:349] Turning on output DP-1
206:41:04.829 [sway/config/output.c:353] Set DP-1 mode to 3840x2160 (-1.000000 Hz)
206:41:04.829 [sway/config/output.c:267] Assigning configured mode to DP-1
206:41:04.829 [sway/config/output.c:327] Output DPI: 162.560000x161.364706
206:41:04.829 [sway/config/output.c:385] Auto-detected output scale: 1.000000
206:41:04.829 [sway/config/output.c:415] Committing output DP-1
206:41:04.829 [INFO] [backend/drm/drm.c:807] Modesetting 'DP-1' with '3840x2160@59997mHz'
206:41:04.829 [DEBUG] [backend/drm/drm.c:689] Initializing renderer on connector 'DP-1'
206:41:04.905 [sway/config/output.c:461] Set DP-1 position to 4480, 0
206:41:04.905 [sway/tree/arrange.c:263] Usable area for ws: 1920x1048@0,32
206:41:04.905 [sway/tree/arrange.c:293] Arranging workspace '1' at 0.000000, 32.000000
206:41:04.905 [sway/tree/arrange.c:263] Usable area for ws: 2560x1440@0,0
206:41:04.905 [sway/tree/arrange.c:293] Arranging workspace '2' at 1920.000000, 0.000000
206:41:04.905 [sway/tree/arrange.c:77] Arranging 0x7fffe17337a0 horizontally
206:41:04.905 [sway/tree/arrange.c:263] Usable area for ws: 3840x2112@0,48
206:41:04.905 [sway/tree/arrange.c:293] Arranging workspace '3' at 4480.000000, 48.000000
206:41:04.905 [sway/tree/arrange.c:156] Arranging 0x7fffe17337a0 vertically
206:41:04.905 [sway/tree/arrange.c:77] Arranging 0x7fffe1733730 horizontally
206:41:04.905 [sway/tree/arrange.c:156] Arranging 0x7fffe17336c0 vertically
206:41:04.906 [sway/tree/arrange.c:77] Arranging 0x7fffe1733650 horizontally
206:41:04.906 [sway/tree/arrange.c:77] Arranging 0x7fffe1733650 horizontally
206:41:04.906 [sway/desktop/transaction.c:427] Transaction 0x2fc5ea0 committing with 18 instructions
206:41:04.906 [sway/desktop/transaction.c:281] Applying transaction 0x2fc5ea0
206:41:04.906 [sway/input/libinput.c:197] sway_input_configure_libinput_device('type:keyboard' on '0:0:Eee_PC_WMI_hotkeys')
206:41:04.933 [sway/input/seat.c:664] Applying input config to 1133:16388:Logitech_K360
206:41:04.933 [sway/input/libinput.c:197] sway_input_configure_libinput_device('type:keyboard' on '1133:16388:Logitech_K360')
206:41:04.947 [sway/input/seat.c:664] Applying input config to 1008:33287:HP_Wireless_Optical_Mobile_Mouse
206:41:04.947 [sway/input/libinput.c:197] sway_input_configure_libinput_device('type:keyboard' on '1008:33287:HP_Wireless_Optical_Mobile_Mouse')
206:41:04.956 [sway/input/seat.c:664] Applying input config to 1133:49257:Logitech_USB_Laser_Mouse
206:41:04.956 [sway/input/libinput.c:197] sway_input_configure_libinput_device('type:keyboard' on '1133:49257:Logitech_USB_Laser_Mouse')
206:41:04.963 [sway/input/seat.c:664] Applying input config to 1118:219:Microsoft_Natural___Ergonomic_Keyboard_4000
206:41:04.963 [sway/input/libinput.c:197] sway_input_configure_libinput_device('type:keyboard' on '1118:219:Microsoft_Natural___Ergonomic_Keyboard_4000')
206:41:04.969 [sway/input/libinput.c:197] sway_input_configure_libinput_device('type:keyboard' on '1118:219:Microsoft_Natural___Ergonomic_Keyboard_4000')
206:41:04.975 [sway/input/libinput.c:197] sway_input_configure_libinput_device('type:keyboard' on '0:3:Sleep_Button')
206:41:04.981 [sway/input/libinput.c:197] sway_input_configure_libinput_device('type:keyboard' on '0:1:Power_Button')
206:41:04.988 [sway/input/libinput.c:197] sway_input_configure_libinput_device('type:keyboard' on '0:6:Video_Bus')
206:41:04.994 [sway/input/libinput.c:197] sway_input_configure_libinput_device('type:keyboard' on '0:1:Power_Button')
206:41:05.000 [sway/config/output.c:349] Turning on output HDMI-A-3
206:41:05.000 [sway/config/output.c:353] Set HDMI-A-3 mode to 2560x1440 (-1.000000 Hz)
206:41:05.000 [sway/config/output.c:267] Assigning configured mode to HDMI-A-3
206:41:05.000 [sway/config/output.c:327] Output DPI: 122.686792x121.920000
206:41:05.000 [sway/config/output.c:385] Auto-detected output scale: 1.000000
206:41:05.000 [sway/config/output.c:415] Committing output HDMI-A-3
206:41:05.000 [INFO] [backend/drm/drm.c:807] Modesetting 'HDMI-A-3' with '2560x1440@59951mHz'
206:41:05.000 [DEBUG] [backend/drm/drm.c:689] Initializing renderer on connector 'HDMI-A-3'
206:41:05.003 [ERROR] [backend/drm/atomic.c:36] HDMI-A-3: Atomic commit failed (modeset): Invalid argument
206:41:05.003 [INFO] [backend/drm/drm.c:727] Page-flip failed with primary FB modifiers enabled, retrying without modifiers
206:41:05.006 [ERROR] [backend/drm/atomic.c:36] HDMI-A-3: Atomic commit failed (modeset): Invalid argument
206:41:05.007 [ERROR] [backend/drm/drm.c:740] Failed to initialize renderer on connector 'HDMI-A-3': initial page-flip failed
206:41:05.007 [ERROR] [backend/drm/drm.c:813] Failed to initialize renderer for plane
206:41:05.007 [sway/config/output.c:420] Failed to commit output HDMI-A-3
206:41:05.007 [sway/config/output.c:349] Turning on output HDMI-A-2
206:41:05.007 [sway/config/output.c:353] Set HDMI-A-2 mode to 1920x1080 (-1.000000 Hz)
206:41:05.007 [sway/config/output.c:267] Assigning configured mode to HDMI-A-2
206:41:05.007 [sway/config/output.c:385] Auto-detected output scale: 1.000000
206:41:05.007 [sway/config/output.c:415] Committing output HDMI-A-2
206:41:05.007 [sway/config/output.c:461] Set HDMI-A-2 position to 0, 0
206:41:05.007 [sway/tree/arrange.c:263] Usable area for ws: 1920x1048@0,32
206:41:05.007 [sway/tree/arrange.c:293] Arranging workspace '1' at 0.000000, 32.000000
206:41:05.007 [sway/tree/arrange.c:263] Usable area for ws: 2560x1440@0,0
206:41:05.007 [sway/tree/arrange.c:293] Arranging workspace '2' at 1920.000000, 0.000000
206:41:05.007 [sway/tree/arrange.c:77] Arranging 0x7fffe17337a0 horizontally
206:41:05.007 [sway/tree/arrange.c:263] Usable area for ws: 3840x2112@0,48
206:41:05.007 [sway/tree/arrange.c:293] Arranging workspace '3' at 4480.000000, 48.000000
206:41:05.007 [sway/tree/arrange.c:156] Arranging 0x7fffe17337a0 vertically
206:41:05.007 [sway/tree/arrange.c:77] Arranging 0x7fffe1733730 horizontally
206:41:05.007 [sway/tree/arrange.c:156] Arranging 0x7fffe17336c0 vertically
206:41:05.007 [sway/tree/arrange.c:77] Arranging 0x7fffe1733650 horizontally
206:41:05.007 [sway/tree/arrange.c:77] Arranging 0x7fffe1733650 horizontally
206:41:05.007 [sway/desktop/transaction.c:427] Transaction 0x241b740 committing with 18 instructions
206:41:05.007 [sway/desktop/transaction.c:281] Applying transaction 0x241b740
206:41:05.007 [sway/input/libinput.c:197] sway_input_configure_libinput_device('type:keyboard' on '0:0:Eee_PC_WMI_hotkeys')
206:41:05.014 [sway/input/seat.c:664] Applying input config to 1133:16388:Logitech_K360
206:41:05.014 [sway/input/libinput.c:197] sway_input_configure_libinput_device('type:keyboard' on '1133:16388:Logitech_K360')
206:41:05.020 [sway/input/seat.c:664] Applying input config to 1008:33287:HP_Wireless_Optical_Mobile_Mouse
206:41:05.020 [sway/input/libinput.c:197] sway_input_configure_libinput_device('type:keyboard' on '1008:33287:HP_Wireless_Optical_Mobile_Mouse')
206:41:05.026 [sway/input/seat.c:664] Applying input config to 1133:49257:Logitech_USB_Laser_Mouse
206:41:05.026 [sway/input/libinput.c:197] sway_input_configure_libinput_device('type:keyboard' on '1133:49257:Logitech_USB_Laser_Mouse')
206:41:05.033 [sway/input/seat.c:664] Applying input config to 1118:219:Microsoft_Natural___Ergonomic_Keyboard_4000
206:41:05.033 [sway/input/libinput.c:197] sway_input_configure_libinput_device('type:keyboard' on '1118:219:Microsoft_Natural___Ergonomic_Keyboard_4000')
206:41:05.039 [sway/input/libinput.c:197] sway_input_configure_libinput_device('type:keyboard' on '1118:219:Microsoft_Natural___Ergonomic_Keyboard_4000')
206:41:05.050 [sway/input/libinput.c:197] sway_input_configure_libinput_device('type:keyboard' on '0:3:Sleep_Button')
206:41:05.060 [sway/input/libinput.c:197] sway_input_configure_libinput_device('type:keyboard' on '0:1:Power_Button')
206:41:05.071 [sway/input/libinput.c:197] sway_input_configure_libinput_device('type:keyboard' on '0:6:Video_Bus')
206:41:05.081 [sway/input/libinput.c:197] sway_input_configure_libinput_device('type:keyboard' on '0:1:Power_Button')
206:41:05.101 [ERROR] [backend/drm/drm.c:364] Failed to page-flip output 'DP-1': a page-flip is already pending
The XKEYBOARD keymap compiler (xkbcomp) reports:
> Warning:          Unsupported maximum keycode 569, clipping.
>                   X11 cannot support keycodes above 255.
> Internal error:   Could not resolve keysym XF86FullScreen
Errors from xkbcomp are not fatal to the X server
The XKEYBOARD keymap compiler (xkbcomp) reports:
> Warning:          Unsupported maximum keycode 569, clipping.
>                   X11 cannot support keycodes above 255.
> Internal error:   Could not resolve keysym XF86FullScreen
Errors from xkbcomp are not fatal to the X server
The XKEYBOARD keymap compiler (xkbcomp) reports:
> Warning:          Unsupported maximum keycode 569, clipping.
>                   X11 cannot support keycodes above 255.
> Internal error:   Could not resolve keysym XF86FullScreen
Errors from xkbcomp are not fatal to the X server
The XKEYBOARD keymap compiler (xkbcomp) reports:
> Warning:          Unsupported maximum keycode 569, clipping.
>                   X11 cannot support keycodes above 255.
> Internal error:   Could not resolve keysym XF86FullScreen
Errors from xkbcomp are not fatal to the X server
The XKEYBOARD keymap compiler (xkbcomp) reports:
> Warning:          Unsupported maximum keycode 569, clipping.
>                   X11 cannot support keycodes above 255.
> Internal error:   Could not resolve keysym XF86FullScreen
Errors from xkbcomp are not fatal to the X server
206:41:05.176 [sway/desktop/layer_shell.c:343] Layer surface destroyed (notifications)
206:41:05.175 [DEBUG] [xwayland/xwm.c:1422] unhandled X11 event: 34
206:41:05.175 [DEBUG] [xwayland/xwm.c:1422] unhandled X11 event: 34
206:41:05.175 [DEBUG] [xwayland/xwm.c:1422] unhandled X11 event: 34
206:41:05.175 [DEBUG] [xwayland/xwm.c:1422] unhandled X11 event: 34
206:41:05.175 [DEBUG] [xwayland/xwm.c:1422] unhandled X11 event: 34
206:41:05.175 [DEBUG] [xwayland/xwm.c:1422] unhandled X11 event: 34
206:41:05.175 [DEBUG] [xwayland/xwm.c:1422] unhandled X11 event: 34
206:41:05.175 [DEBUG] [xwayland/xwm.c:1422] unhandled X11 event: 34
206:41:05.175 [DEBUG] [xwayland/xwm.c:1422] unhandled X11 event: 34
206:41:05.175 [DEBUG] [xwayland/xwm.c:1422] unhandled X11 event: 34
The XKEYBOARD keymap compiler (xkbcomp) reports:
> Warning:          Unsupported maximum keycode 569, clipping.
>                   X11 cannot support keycodes above 255.
> Internal error:   Could not resolve keysym XF86FullScreen
Errors from xkbcomp are not fatal to the X server
206:41:05.190 [DEBUG] [xwayland/xwm.c:1422] unhandled X11 event: 34
206:41:05.190 [DEBUG] [xwayland/xwm.c:1422] unhandled X11 event: 34
The XKEYBOARD keymap compiler (xkbcomp) reports:
> Warning:          Unsupported maximum keycode 569, clipping.
>                   X11 cannot support keycodes above 255.
> Internal error:   Could not resolve keysym XF86FullScreen
Errors from xkbcomp are not fatal to the X server
206:41:05.200 [DEBUG] [xwayland/xwm.c:1422] unhandled X11 event: 34
206:41:05.200 [DEBUG] [xwayland/xwm.c:1422] unhandled X11 event: 34
The XKEYBOARD keymap compiler (xkbcomp) reports:
> Warning:          Unsupported maximum keycode 569, clipping.
>                   X11 cannot support keycodes above 255.
> Internal error:   Could not resolve keysym XF86FullScreen
Errors from xkbcomp are not fatal to the X server
206:41:05.212 [DEBUG] [xwayland/xwm.c:1422] unhandled X11 event: 34
206:41:05.212 [DEBUG] [xwayland/xwm.c:1422] unhandled X11 event: 34
The XKEYBOARD keymap compiler (xkbcomp) reports:
> Warning:          Unsupported maximum keycode 569, clipping.
>                   X11 cannot support keycodes above 255.
> Internal error:   Could not resolve keysym XF86FullScreen
Errors from xkbcomp are not fatal to the X server
206:41:05.220 [DEBUG] [xwayland/xwm.c:1422] unhandled X11 event: 34
206:41:05.220 [DEBUG] [xwayland/xwm.c:1422] unhandled X11 event: 34
The XKEYBOARD keymap compiler (xkbcomp) reports:
> Warning:          Unsupported maximum keycode 569, clipping.
>                   X11 cannot support keycodes above 255.
> Internal error:   Could not resolve keysym XF86FullScreen
Errors from xkbcomp are not fatal to the X server
206:41:05.229 [DEBUG] [xwayland/xwm.c:1422] unhandled X11 event: 34
206:41:05.229 [DEBUG] [xwayland/xwm.c:1422] unhandled X11 event: 34
The XKEYBOARD keymap compiler (xkbcomp) reports:
> Warning:          Unsupported maximum keycode 569, clipping.
>                   X11 cannot support keycodes above 255.
> Internal error:   Could not resolve keysym XF86FullScreen
Errors from xkbcomp are not fatal to the X server
206:41:05.238 [DEBUG] [xwayland/xwm.c:1422] unhandled X11 event: 34
206:41:05.238 [DEBUG] [xwayland/xwm.c:1422] unhandled X11 event: 34
The XKEYBOARD keymap compiler (xkbcomp) reports:
> Warning:          Unsupported maximum keycode 569, clipping.
>                   X11 cannot support keycodes above 255.
> Internal error:   Could not resolve keysym XF86FullScreen
Errors from xkbcomp are not fatal to the X server
206:41:05.244 [DEBUG] [xwayland/xwm.c:1422] unhandled X11 event: 34
206:41:05.244 [DEBUG] [xwayland/xwm.c:1422] unhandled X11 event: 34
The XKEYBOARD keymap compiler (xkbcomp) reports:
> Warning:          Unsupported maximum keycode 569, clipping.
>                   X11 cannot support keycodes above 255.
> Internal error:   Could not resolve keysym XF86FullScreen
Errors from xkbcomp are not fatal to the X server
206:41:05.250 [DEBUG] [xwayland/xwm.c:1422] unhandled X11 event: 34
206:41:05.250 [DEBUG] [xwayland/xwm.c:1422] unhandled X11 event: 34
The XKEYBOARD keymap compiler (xkbcomp) reports:
> Warning:          Unsupported maximum keycode 569, clipping.
>                   X11 cannot support keycodes above 255.
206:41:05.254 [DEBUG] [types/wlr_surface.c:666] New wlr_surface 0x2e14570 (res 0x2f23590)
206:41:05.254 [DEBUG] > [types/xdg_shell/wlr_xdg_surface.c:462] new xdg_surface 0x23fdca0 (res 0x2d5bf20)Internal error:
Could not resolve keysym XF86FullScreen
206:41:05.255 [sway/desktop/xdg_shell.c:514] New xdg_shell popup
Errors from xkbcomp are not fatal to the X server
206:41:05.256 [DEBUG] [xwayland/xwm.c:1422] unhandled X11 event: 34
206:41:05.256 [DEBUG] [xwayland/xwm.c:1422] unhandled X11 event: 34
The XKEYBOARD keymap compiler (xkbcomp) reports:
> Warning:          Unsupported maximum keycode 569, clipping.
>                   X11 cannot support keycodes above 255.
> Internal error:   Could not resolve keysym XF86FullScreen
Errors from xkbcomp are not fatal to the X server
206:41:05.260 [DEBUG] [xwayland/xwm.c:1422] unhandled X11 event: 34
206:41:05.260 [DEBUG] [xwayland/xwm.c:1422] unhandled X11 event: 34
The XKEYBOARD keymap compiler (xkbcomp) reports:
> Warning:          Unsupported maximum keycode 569, clipping.
>                   X11 cannot support keycodes above 255.
> Internal error:   Could not resolve keysym XF86FullScreen
Errors from xkbcomp are not fatal to the X server
206:41:05.266 [DEBUG] [xwayland/xwm.c:1422] unhandled X11 event: 34
206:41:05.266 [DEBUG] [xwayland/xwm.c:1422] unhandled X11 event: 34
The XKEYBOARD keymap compiler (xkbcomp) reports:
> Warning:          Unsupported maximum keycode 569, clipping.
>                   X11 cannot support keycodes above 255.
> Internal error:   Could not resolve keysym XF86FullScreen
Errors from xkbcomp are not fatal to the X server
206:41:05.272 [DEBUG] [xwayland/xwm.c:1422] unhandled X11 event: 34
206:41:05.272 [DEBUG] [xwayland/xwm.c:1422] unhandled X11 event: 34
The XKEYBOARD keymap compiler (xkbcomp) reports:
> Warning:          Unsupported maximum keycode 569, clipping.
>                   X11 cannot support keycodes above 255.
> Internal error:   Could not resolve keysym XF86FullScreen
Errors from xkbcomp are not fatal to the X server
206:41:05.277 [DEBUG] [xwayland/xwm.c:1422] unhandled X11 event: 34
206:41:05.277 [DEBUG] [xwayland/xwm.c:1422] unhandled X11 event: 34
The XKEYBOARD keymap compiler (xkbcomp) reports:
> Warning:          Unsupported maximum keycode 569, clipping.
>                   X11 cannot support keycodes above 255.
> Internal error:   Could not resolve keysym XF86FullScreen
Errors from xkbcomp are not fatal to the X server
206:41:05.282 [DEBUG] [xwayland/xwm.c:1422] unhandled X11 event: 34
206:41:05.282 [DEBUG] [xwayland/xwm.c:1422] unhandled X11 event: 34
The XKEYBOARD keymap compiler (xkbcomp) reports:
> Warning:          Unsupported maximum keycode 569, clipping.
>                   X11 cannot support keycodes above 255.
> Internal error:   Could not resolve keysym XF86FullScreen
Errors from xkbcomp are not fatal to the X server
206:41:05.287 [DEBUG] [xwayland/xwm.c:1422] unhandled X11 event: 34
206:41:05.287 [DEBUG] [xwayland/xwm.c:1422] unhandled X11 event: 34
The XKEYBOARD keymap compiler (xkbcomp) reports:
> Warning:          Unsupported maximum keycode 569, clipping.
>                   X11 cannot support keycodes above 255.
> Internal error:   Could not resolve keysym XF86FullScreen
Errors from xkbcomp are not fatal to the X server
206:41:05.292 [DEBUG] [xwayland/xwm.c:1422] unhandled X11 event: 34
206:41:05.292 [DEBUG] [xwayland/xwm.c:1422] unhandled X11 event: 34
206:41:06.108 [DEBUG] [backend/session/session.c:45] udev event for card0 (change)
206:41:06.108 [DEBUG] [backend/drm/backend.c:112] /dev/dri/card0 invalidated
206:41:06.108 [INFO] [backend/drm/drm.c:1252] Scanning DRM connectors
206:41:06.108 [INFO] [backend/drm/drm.c:1427] 'DP-1' disconnected
206:41:06.108 [INFO] [backend/drm/drm.c:1617] Emitting destruction signal for 'DP-1'
206:41:06.108 [DEBUG] [backend/drm/drm.c:1058] De-allocating CRTC 0 for output 'DP-1'
206:41:06.139 [sway/tree/output.c:264] Disabling output 'DP-1'
206:41:06.140 [DEBUG] [types/wlr_output.c:1049] Falling back to software cursor on output 'DP-1'
[285922:285922:1111/095741.558311:ERROR:x11_error_handler.cc(63)] X IO error received (X server probably went away)
```

**Comment by hypergig** [comment]
When I reload my config I the same fire hose of:

```
Errors from xkbcomp are not fatal to the X server
The XKEYBOARD keymap compiler (xkbcomp) reports:
> Warning:          Unsupported maximum keycode 569, clipping.
>                   X11 cannot support keycodes above 255.
```
Most times it completely crashes sway, not sure it is related but I assume so as reloading config does destroy outputs.

**Comment by danieldg** [comment]
@hypergig: I am pretty sure the xkbcomp stuff is completely unrelated to this bug; I just didn't cut it out of the log.

You should get the backtrace and compare it to see if you are also observing this bug, or if it's a different one.

**Comment by arbrauns** [comment]
Probably fixed by #6820.

**Comment by emersion** [comment]
Yup, should be fixed by that PR indeed. Please re-open if that's not the case.
